pkg/template/resource_test.go

```diff
@@ -320,3 +321,13 @@ func Test_ResolveBinding_error(t *testing.T) {
 		})
 	}
 }
+
+func Test_ApplyUIDToResourceTemplate(t *testing.T) {
+	uid := Uid()
+	rt := json.RawMessage(`{"name": "something-$(uid)"}`)
+	expectedRt := bytes.Replace(rt, uidMatch, []byte(uid), -1)
```

Comment:
Can you please write the `expectedRt` as a string like this:
```
json.RawMessage(`{"name": "something-dpms2"}`)
or maybe this:
json.RawMessage(fmt.Sprintf(`{"name": "something-$(%s)"}`, uid))
```
You probably shouldn't be verifying that your test works by copying the function's implementation code in your test case 😄  (`return bytes.Replace(rt, uidMatch, []byte(uid), -1)`)

Reply:
For sure, this was before I seeded it. Forgot to change it.